docs/installing-northstar/troubleshooting.md

```diff
@@ -89,10 +89,15 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
 
 Go to your `Titanfall2` directory. From there go to the `R2Northstar` and delete `enabledmods.json`. This file stores information about which mods are enabled and disabled. By deleting this file Northstar will fall back to the default (all mods enabled) and re-create the file appropriately.
 
-## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
+## MSVCR120.dll / MSVCP120.dll Not found <a href="#msvcr" id="msvcr"></a>
+
+If you get this error you can fix it by [downloading and installing vcredist 2013 (`vcredist_x64.exe`)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
+
+![MSVCR120.dll Error](../images/msvcr.png)
 
-If you get this error you can fix it by [installing vcredist 2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If it does not work install the 2013 version
+## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
 
+If you get this error you can fix it by [installing vcredist 2015-2022 (`vcredist_x64.exe`)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022). If it does not work install the 2013 version
 ## Launch Northstar Locked <a href="#locked-northstar" id="locked-northstar"></a>
```

Comment:
```suggestion

## Launch Northstar Locked <a href="#locked-northstar" id="locked-northstar"></a>
```

Just noticed a missing newline here. Might have introduced by a suggested change of mine, whoops. I'll fix it ^^